src/internal/Esp32_i2s.c

```diff
@@ -15,12 +15,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#if defined(ARDUINO_ARCH_ESP32) 
-
-#include "sdkconfig.h" // this sets useful config symbols, like CONFIG_IDF_TARGET_ESP32C3
-
-// ESP32C3/S3 I2S is not supported yet due to significant changes to interface
-#if !defined(CONFIG_IDF_TARGET_ESP32C3) && !defined(CONFIG_IDF_TARGET_ESP32S3)
+#if defined(ARDUINO_ARCH_ESP32)
```

Comment:
we probably dont need this file. Its support for the Espressif ESP32 processor 
https://www.espressif.com/en/products/socs/esp32

It is interesting to see that they use the I2S line of the processor, instead of the spi